src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int EMPTY_STACK_VALUE = -1;
+	private static final int FULL_STACK_VALUE = 11;
+	private static final int STACK_CAPACITY = 12;
 
-	public int totalCount = -1;
+	private final int[] numbers = new int[STACK_CAPACITY];
 
-	public void countIn(int newValue) {
+	private int totalCount = EMPTY_STACK_VALUE;

```

Comment:
Attribute name incorrect.
Extracting this constant is done incorrectly. Should show during next classes ;)